### ISSUE TITLE
Add Environmental Considerations to Mask Advice?

### ISSUE THREAD

**Issue by TheSre** [microcovid/microcovid#958]
Hi,
Thank you so much for the microCOVID project! I think it's really valuable and a very helpful resource! One thing I wanted to surface for consideration regards the advice for masks. After thinking and reading (e.g. [this article](https://theconversation.com/coronavirus-face-masks-an-environmental-disaster-that-might-last-generations-144328)) more about the environmental impact of disposable masks, I'm a bit concerned that more people are using disposable masks than is necessary. I'm not very educated on how this environmental harm might stack up against the benefit of  better protection of certain disposable masks, but I just wanted to bring it up as food for thought in terms of how you are presenting your mask advice.
Thank you for your work, stay safe, and enjoy your day!

**Comment by beshaya** [comment]
Yeah, I'm also really sad about how often I see discarded masks on the street. I don't know how to convince people to care 😢. Just because masks are disposable, doesn't mean you have to toss it daily (I use my KN95's for a month)

In terms of impact on landfill / petroleum use for masks that are properly disposed of I suspect, a mask per day is relatively little waste compared to what the average American uses from plastic packaging from the supermarket / takeout / etc. The EPA says Americans throw out 35 million tons of plastic every year. That's about 0.6 pounds of plastic per person per day. A surgical mask weighs 0.007 pounds, so maybe represents a 1% increase in waste if everyone disposed of one every day. Of course, people should reduce waste if it's easy, but if I were going to pick an environmental program to champion, this wouldn't be at the top of the list.

**Comment by TheSre** [comment]
> Yeah, I'm also really sad about how often I see discarded masks on the street. I don't know how to convince people to care cry. Just because masks are disposable, doesn't mean you have to toss it daily (I use my KN95's for a month)
> 
> In terms of impact on landfill / petroleum use for masks that are properly disposed of I suspect, a mask per day is relatively little waste compared to what the average American uses from plastic packaging from the supermarket / takeout / etc. The EPA says Americans throw out 35 million tons of plastic every year. That's about 0.6 pounds of plastic per person per day. A surgical mask weighs 0.007 pounds, so maybe represents a 1% increase in waste if everyone disposed of one every day. Of course, people should reduce waste if it's easy, but if I were going to pick an environmental program to champion, this wouldn't be at the top of the list.

Thanks for the response! That totally makes sense and I also suspect that disposed masks make up a pretty small part of the average person's impact on the environment. One thing that came to mind after reading what you wrote: I wonder how much weight tells us about environmental impact. Maybe lighter things could have a large impact if they are prone to break down into microplastics more easily, which could then pollute groundwater, water streams, etc. Anyway, that's getting pretty speculative and maybe less relevant--the main thing I'm getting at is that it doesn't seem obvious that widespread disposable mask usage has a trivial impact on the environment.

**Comment by beshaya** [comment]
For stuff that gets littered, I totally agree that small + light items can
be pretty bad; surface area might be a better measurement for these. I just
don't think the people who toss their masks on the street care about the
environment, so I doubt a post on microCOVID would change their behavior. I
can't imagine the kind of person who casually throws their trash on the
street would change their behavior if you told them to buy a different
mask. Outside the scope of COVID, I'd be fascinated to know why people toss
their trash on the street instead of walking a few feet to a trash can and
how to change those behaviors.

For things that get put in the trash, compacted, and end up in landfill
(which I imagine is where a disposable mask that you or I wear would end
up), I think weight is a pretty good proxy for impact.

What we really want is for 3M to make biodegradable disposable surgical +
N95 masks. But that seems like a pipe dream at this point.
